### PR TITLE
Add more examples, add missing bracket, clean up URLs

### PR DESCRIPTION
**Pull request by vedranmiletic**
Subject: Minor fixes and a couple of additions to EXAMPLES

### Feature or Bugfix
- Feature
- Bugfix

### Detail
- More examples added
- Added one missing closing bracket
- Cleaned up two URLs

### CODE REVIEW

File: EXAMPLES [hunk 1]
```diff
@@ -55,6 +56,8 @@ Documentation using the classic theme
 * `Buildbot <https://docs.buildbot.net/latest/>`__
 * `CMake <https://cmake.org/documentation/>`__ (customized)
 * `Chaco <https://docs.enthought.com/chaco/>`__ (customized)
+* `CORE <https://downloads.pf.itd.nrl.navy.mil/docs/core/core-html/>`__
+* `CORE Python modules <https://downloads.pf.itd.nrl.navy.mil/docs/core/core-python-html/>__
```
Comment (tk0miya) [inline EXAMPLES]: It seems this mark up has been broken (not closed).

**Review by tk0miya — COMMENTED** [review]
LGTM with nits

File: EXAMPLES (hunk 1 above)
Comment (vedranmiletic) [reply EXAMPLES]: Whoops, fixed.

**Review by tk0miya — APPROVED** [review]
Confirmed!

**Comment by tk0miya** [comment]
Thank you for your contribution!

**Pull request merged**